src/pages/AddPage/AddPage.jsx

```diff
@@ -1,7 +1,46 @@
 import React from "react";
+import "./AddPage.scss";
+import { useState } from "react";
 
 const AddPage = () => {
-  return <h2>add</h2>;
+  const url = "http://localhost:3030";
+  const [text, setText] = useState("");
+  const [error, setError] = useState(null);
+  const [done, setDone] = useState(null);
```

Comment:
Add a new line after this to separate the function below with the states definitions.

Reply:
Done.